docs/reference/troubleshooting/common-issues/task-queue-backlog.asciidoc

```diff
@@ -1,51 +1,90 @@
 [[task-queue-backlog]]
 === Task queue backlog
 
-A backlogged task queue can prevent tasks from completing and 
-put the cluster into an unhealthy state. 
-Resource constraints, a large number of tasks being triggered at once,
-and long running tasks can all contribute to a backlogged task queue.
+A backlogged task queue can prevent tasks from completing and put the cluster
+into an unhealthy state. Resource constraints, a large number of tasks being
+triggered at once, and long running tasks can all contribute to a backlogged
+task queue.
 
 [discrete]
 [[diagnose-task-queue-backlog]]
 ==== Diagnose a task queue backlog
 
 **Check the thread pool status**
 
-A <<high-cpu-usage,depleted thread pool>> can result in <<rejected-requests,rejected requests>>. 
+A <<high-cpu-usage,depleted thread pool>> can result in
+<<rejected-requests,rejected requests>>. This may surface restricted to a
+<<data-tiers,data tier>>'s traffic, potentially with <<hotspotting,hot spotting>>
+symptoms.
```

Comment:
a similar sentiment was confusing on the other page and prob needs a similar solution. took some liberties here and perhaps injected some incorrect content

```suggestion
<<rejected-requests,rejected requests>>. 

Thread pool depletion might be restricted to a specific <<data-tiers,data tier>> if that tier, and therefore the nodes assigned to that tier, is experiencing more traffic than other tiers. This imbalance in resource utilization is also known as <<hotspotting,hot spotting>>.
```

Reply:
The Hot Spotting Elastic discusses or is concerned about is node(s) within a data tier not across data tiers. Lower data tiers are generally expected to be cold-spotted compared to earlier tiers.

Reply:
Sorry I realize my statement was confusing but don't know how to help myself. 

An example of a depleted thread pool which is expected is that the majority of write requests process into hot nodes. If bulk requests are spread even across the hot nodes, it is still possible to fully fill up their queue. This is what "thread pool depletion" was trying to call out before me.  However, something Support sees frequently is when hot spotting does occur, a node within the hot tier gets depleted while the others don't. This still slows ingest and/or potentially cluster & needs to be looked at / resolved. In this example, the problem happens only within hot tier and we would not care how warm/cold/frozen was doing compared to hot because the work they do is different.